client/.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": ["eslint:recommended", "plugin:react/recommended", "prettier", "plugin:@typescript-eslint/recommended"],
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": ["react", "@typescript-eslint"],
+    "parser": "@typescript-eslint/parser",
+    "rules": {
+        "indent": ["warn", "tab"],
+        "quotes": ["error", "single"],
+        "semi": ["error", "always"],
+        "no-unused-vars": "off",
```

Comment:
I see no point in keeping unused variables, can we just enable this rule?

Reply:
It doubles "@typescript-eslint/no-unused-vars": ["error"],

Reply:
Ah it makes sense, thanks